Tasks/CordovaBuild/node-setup.js

```diff
@@ -4,32 +4,52 @@
 */
 
 var path = require('path'),
+    fs = require('fs'),
     semver = require('semver'),
     nodeManager = require('./lib/node-manager.js'),
     taskLibrary = require('./lib/vso-task-lib-proxy.js'),
     teambuild = require('taco-team-build');
 
 var nodeSetupPromise;
+
+var cwd = taskLibrary.getInput('cwd', /* required */ false);
+var tacoFile = path.join((cwd ? cwd : '.'), 'taco.json');
+
 var version = taskLibrary.getInput('cordovaVersion', /* required */ false);
+if (!version) {
+    try {
+        var stats = fs.statSync(tacoFile);
+        if (stats && stats.isDirectory()) {
+            version = require(tacoFile)['cordova-cli'];
+            console.log('Cordova version set to ' + version + ' based on the contents of taco.json');
+        }
+    } catch (e) { }
+}
+
```

Comment:
There is one more fallback we need to consider unfortunately - that's if an environment variable is set.  The variable varies by task name. 

CORDOVA_DEFAULT_VERSION

or

IONIC_DEFAULT_VERSION

or 

PHONEGAP_DEFAULT_VERSION

So, order of precedence is:

1. Cordova version specified by task
2. Cordova version in taco.json
3. Env var
4. else edge

It's the same logic that's in taco-team-build - we should consider rationalizing it when we pick up the other refactor task to reduce risk of breaking if vso-task-lib drops support for 0.12.7